module/zstd/Makefile.in

```diff
@@ -0,0 +1,47 @@
+src = @abs_top_srcdir@/module/zstd
+obj = @abs_builddir@
+
+MODULE := zzstd
+
+obj-$(CONFIG_ZFS) := $(MODULE).o
+
+ZFS_MODULE_CFLAGS += -I@abs_top_srcdir@/contrib/zstd/zstd-1.4.4/lib
+ZFS_MODULE_CFLAGS += -I@abs_top_srcdir@/contrib/zstd/zstd-1.4.4/lib/common
+ZFS_MODULE_CFLAGS += -I@abs_top_srcdir@/contrib/zstd/zstd-1.4.4/lib/compress
+ZFS_MODULE_CFLAGS += -I@abs_top_srcdir@/contrib/zstd/zstd-1.4.4/lib/decompress
+ZFS_MODULE_CFLAGS += -I@abs_top_srcdir@/include/sys/zstd
+
+asflags-y += $(ZFS_MODULE_CFLAGS)
+ccflags-y += $(ZFS_MODULE_CFLAGS) $(ZFS_MODULE_CPPFLAGS)
+
+# Suppress unused but set variable warnings often due to ASSERTs
+ccflags-y += $(NO_UNUSED_BUT_SET_VARIABLE)
+# frame-larger-than=20480 is needed here due to requirements of the stock zstd
+ccflags-y += -Wframe-larger-than=20480 -O3
```

Comment:
typo. must be 2048 like in my tree. it has no negative consequences but its overdosed

Reply:
original code 
`# Suppress unused but set variable warnings often due to ASSERTs
ccflags-y += $(NO_UNUSED_BUT_SET_VARIABLE)

ccflags-y += -Wframe-larger-than=2048 -O3
`

Reply:
@brainslayer
No, please read back your logs.
It used to be 20480 and it got set back to 2048 because you cut out the code from ZSTD requiring the 20480.

Reply:
I'm not very inclined to read everything again to quote it to  you tbh.
And we also tested this a few days back afaik.

Reply:
as I already meantioned, it must be 2048***0***, because I get something like 16k

Reply:
2048 is the value to be used. trust me. i tested it. the zstd code is wrong if such a value is used in that library. and 20480 looks really like just a typo. just a zero too much. and this value does also just suppress warnings. it has no effect how the compiler does its job

Reply:
@brainslayer you tested it before, and concluded 20480 was required if using the unaltered zstd library. Those are YOUR advices.

In the link i send to you before, you yourself stated that it was "now" possible to use 2048 again, after changes you did to ZSTD.

Reply:
uargh! this is NOT A TYPO! I - myself - changed it to 2048!

Reply:
@c0d3z3r0 Now **thats** a type... you yourself changed it to 2048**0**

Reply:
@Ornias1993 indeed.

Reply:
@c0d3z3r0 As stated in the other thread:
> @c0d3z3r0 To prevent endless discussions with reviewers, lets make it like this:
> `# Due to stock zstd using some 16k frames frame-larger-than=20480 is required`
> 
> This also gives the literal reason WHY it is required for ZSTD stock ;)



https://github.com/zfsonlinux/zfs/pull/9655#discussion_r353280814

Reply:
@brainslayer: If you can do an debug install of this codebase with 2048 and it gives no errors, please do comment about it.

But after you actually tried it, on this codebase, okey?